Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx

```diff
@@ -457,6 +462,39 @@ int vtkMRMLAbstractViewNode::GetRulerTypeFromString(const char* name)
   return -1;
 }
 
+//-----------------------------------------------------------
+const char* vtkMRMLAbstractViewNode::GetRulerColorAsString(int id)
+{
+  switch (id)
+    {
+    case RulerColorWhite: return "white";
+    case RulerColorBlack: return "black";
+    default:
+      // invalid id
+      return "";
+    }
+}
+
+//-----------------------------------------------------------
+int vtkMRMLAbstractViewNode::GetRulerColorFromString(const char* name)
+{
+  if (name == nullptr)
+  {
```

Comment:
Indentation

Reply:
I agree it's a bit odd, but this matches the style for GetRulerTypeAsString above so I figured it's better to stick with precedent.

Reply:
Ah, I see,  you were talking about GetRulerColorFromString - yes, that's an indent issue I copied from GetRulerTypeFrom string - I'll fix them both.